packages/nimble-components/src/rich-text-viewer/template.ts

```diff
@@ -2,5 +2,7 @@ import { html } from '@microsoft/fast-element';
 import type { RichTextViewer } from '.';
 
 export const template = html<RichTextViewer>`
-    <template> Rich Text Viewer here. </template>
+    <template>
+        <div class="container" id="viewer"></div>
```

Comment:
We don't want to give any elements in our template an actual `id`, as they need to be unique values on a page, and should generally be used by app developers. To access this part in your test code, you can just use the `class` value.